### ISSUE TITLE
Alarm does not wake up screen

### ISSUE THREAD

**Issue by lutin11** [ubports/ubuntu-touch#1571]
- Device: Nexus 5
- Channel: stable
- Build: OTA-13

### Steps to reproduce
Setup alarm
Alarm ring up but screen doesn't wake-up

### Expected behavior
Alarm ring and screen wake-up

### Logfiles and additional information
media-hub.log: https://paste.ubuntu.com/p/pqVbqMTsMZ/
unity8.log: https://paste.ubuntu.com/p/S4fgpNBgH4/

**Comment by Fuseteam** [comment]
seems specific for the nexus 5, cannot reproduce this with the e5

**Comment by lutin11** [comment]
I've got same problem on devel channel but I do not have the problem on OnePlus One

**Comment by Fuseteam** [comment]
that only seems to confirm the issue for the nexus 5

**Comment by UniversalSuperBox** [comment]
Hello @lutin11,

Is this problem still occurring for you on OTA-14? Is your device connected to a mobile network or WiFi when it occurs?

**Comment by lutin11** [comment]
> Hello @lutin11,
> 
Hello @UniversalSuperBox 
> Is this problem still occurring for you on OTA-14? Is your device connected to a mobile network or WiFi when it occurs?
I'am currently running last devel version.
The phone is on plane mode when alarm ring-up, but I've tried with network on and It's same behaviours.

**Comment by Fuseteam** [comment]
so on latest devel the alarm doesn't wake the screen whether on airplane mode or not?

**Comment by lutin11** [comment]
> so on latest devel the alarm doesn't wake the screen whether on airplane mode or not?
Yes I confirm, on latest devel the alarm does not wake up the screen  on airplane mode as not in airplane mode

**Comment by lutin11** [comment]
when alarm ring-up, I've got the followings logs in /var/log/syslog

`==> /var/log/syslog <==
Nov 23 21:40:01 ubuntu-phablet kernel: [ 1973.322222] adm_callback: cmd = 0x10328 returned error = 0x2
Nov 23 21:40:01 ubuntu-phablet kernel: [ 1973.322874] adm_callback: cmd = 0x10328 returned error = 0x2
`

**Comment by UniversalSuperBox** [comment]
`handle_proximity_event(near)` seems prescriptive in this case. Is there something on top of your Nexus 5, or a screen protector that may be interfering with the proximity sensor, when this issue occurs? Are you able to make phone calls normally, with your device shutting its screen off when it is lifted to your face?
